gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -124,9 +130,20 @@ def main(args):
                 "Adding VEP context annotations and preparing tables for constraint"
                 " calculations..."
             )
-            # TODO: Need to add function that annotates methylation, coverage, and
-            #  gerp in the vep context table.
-            context_ht = annotated_context_ht.versions[version].ht()
+            # Annotates methylation, coverage, and gerp in the vep context Table.
+            if prepare_context_ht:
+                split_context_ht(
+                    vep_context.versions["101"].ht(),
+                    {
+                        "exomes": get_coverage_ht("exomes").ht(),
+                        "genomes": get_coverage_ht("genomes").ht(),
+                    },
+                    methylation_ht.versions[version].ht(),
+                    gerp_ht.versions[version].ht(),
+                ).write(get_annotated_context_ht(version).path, overwrite)
+                context_ht = get_annotated_context_ht(version).ht()
+            else:
+                context_ht = get_annotated_context_ht(use_old_data=True).ht()
```

Comment:
We might want to do the same thing I suggested for the mutation rate HT here, so add an argument and move `if prepare_context_ht:` outside of `if args.preprocess_data:`  then load the `context_ht` based on the argument